cfgov/unprocessed/css/organisms/post-preview.less

```
@@ -102,20 +135,48 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left,
+        &_right {
+            display: inline-block;
+            float: none;
+        }
+
+        &_left {
+            .heading-4();
+            line-height: unit( 24px / 18px );
+
+            .m-meta-header_separator {
+                padding-left: 0.25em;
+                padding-right: 0.25em;
+            }
+        }
+
+        &_right {
+            line-height: 1.5;
+
+            .a-date {
+                line-height: unit( 24px / 14px );
+            }
+        }
+    }
+
+    &_content {
+        grid-area: content;
+        padding-right: unit( @grid_gutter-width / @base-font-size-px, em );
     }
 
     &_description {
         margin-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em);
     }
 
     &_image-container {
+        grid-area: image;
+        width: unit( 270px / @base-font-size-px, em);
```

The `width` property and I assume also `grid-area` need to remain in a media query for images to be full-width on XS and S displays. A comment explaining the magic number would also be helpful.

```suggestion
        width: unit( 270px / @base-font-size-px, em );
```